RDSC-1311 Additional info about Gears on K8s (#2600)
RedisLabs/redislabs-docs: content/rdi/installation/install-k8s.md

## Prerequisites

> - An existing Redis Enterprise cluster version >= 6.2.18

Reviewer: consider having the 6.2.18 as a hugo parameter as it's probably used elsewhere throughout RDI documentation.

Author: it's the minimal version of this API so won't change?

> - An existing Redis Enterprise cluster version >= 6.2.18

Reviewer: We can still install it on >= 6.2 and < 6.2.18, no? just manual instructions should be embraced.

Author: you can't install with the old API on REC

> - [RedisGears](https://redis.com/modules/redis-gears/) {{<param rdi_redis_gears_current_version>}} installed on the cluster. In case it's missing, see [Install RedisGears for Redis Data Integration]({{<relref "/rdi/installation/install-redis-gears.md">}}) to install. 

Reviewer: remove redundant space after the period.

Author: done

- A target Redis DB (can be added after installation).

> > Note: The Gears binaries to install must match the base OS of RE containers. In case of [Rancher](https://www.rancher.com/), REC base OS is Ubuntu 18.04. Use the following command to install Gears

Author: done

> > Note: The Gears binaries to install must match the base OS of REC containers. In case of [Rancher](https://www.rancher.com/), REC base OS will be Ubuntu 18.04. Use the following command to install Gears

Reviewer: add period for the second sentence.

> > Note: The Gears binaries to install must match the base OS of REC containers. In case of [Rancher](https://www.rancher.com/), REC base OS will be Ubuntu 18.04. Use the following command to install Gears

Reviewer: I think this is better, but you decide:
For [Rancher](https://www.rancher.com/), the REC base OS is Ubuntu 18.04.

Author: done

> > Note: The Gears binaries to install must match the base OS of REC containers. In case of [Rancher](https://www.rancher.com/), REC base OS will be Ubuntu 18.04. Use the following command to install Gears

Reviewer: Gears --> RedisGears (2 occurrences)

Author: done

> > Note: The Gears binaries to install must match the base OS of REC containers. In case of [Rancher](https://www.rancher.com/), REC base OS will be Ubuntu 18.04. Use the following command to install Gears

Reviewer: REC - not sure if it's common that it stands for Redis Cluster. Either mention it in brackets the first time you refer to Redis Cluster here:
```
An existing Redis Enterprise cluster version >= 6.2.18
```

Or just use Redis Enterprise cluster here instead.

Author: done

```bash
curl -s https://redismodules.s3.amazonaws.com/redisgears/redisgears.Linux-ubuntu18.04-x86_64.1.2.6-withdeps.zip -o /tmp/redis-gears.zip
```

> In case the wrong Gears binaries had been installed, use the following commands to fix it

Reviewer: Gears --> Redis Gears

Author: done

Reviewer: Redis Gears --> RedisGears

> In case the wrong Gears binaries had been installed, use the following commands to fix it

Reviewer: Add :

Author: done

```bash
# start port forwarding to the rec api
kubectl port-forward service/rec 9443:9443

> # find the uid of the gears module

Reviewer: gears --> RedisGears

Author: done

Reviewer: Redis Gears --> RedisGears

Author: done

# note: skip piping to the jq if it is not installed

> curl -k -v -u "user:pwd" https://localhost:9443/v1/modules | jq '.[] | {module_name,uid,semantic_version}'

Reviewer: jq might be missing, should be installed beforehand

Author: he has a comment there

> # put the gears module uid instead of <uid>

Reviewer: gears --> RedisGears

Author: done

curl -k -s -u "user:pwd" -X DELETE https://localhost:9443/v2/modules/<uid>

> # install the correct version of the gears module

Reviewer: gears --> RedisGears

Author: done

curl -k -s -u "user:pwd" -F "module=@/tmp/redis-gears.zip" https://localhost:9443/v2/modules

# check the version of the newly installed module

> curl -k -v -u "user:pwd" https://localhost:9443/v1/modules | jq '.[] | {module_name,uid,semantic_version}'

Author: done

  You should get a message - "Successfully configured redis-di instance on port <REDIS_DI_PORT>"

> The `create` command will create a BDB named `redis-di-1` in your cluster. You will need to use a privileged Redis Enterprise user that has the permissions to create a BDB and to register RedisGears recipes, to run it.

Reviewer 2: ```suggestion
The `create` command will create a database named `redis-di-1` in your cluster. You must use a privileged Redis Enterprise user with the necessary permissions to create a database and register RedisGears recipes to run it.
```
